src/OrbitGl/TextRenderer.h

```diff
@@ -61,6 +61,9 @@ class TextRenderer {
   [[nodiscard]] float GetStringWidth(const char* text, uint32_t font_size);
   [[nodiscard]] float GetStringHeight(const char* text, uint32_t font_size);
 
+  void PushTranslation(float x, float y, float z = 0.f) { translations_.PushTranslation(x, y, z); }
+  void PopTranslation() { translations_.PopTranslation(); }
+
```

Comment:
This is not needed, is it?

Reply:
These will be used as public API functions later. I chose to not expose `translations_` to the outside and provide those two methods instead.